week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -15,33 +15,57 @@
   1. This program throws an error. Why? (If you can't find it, try executing it).
   2. Remove the line that throws the error.
   3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
+  4. How many times is "f2" called?-------3 times when value of i is 0 2 and 4 and so on basically when remainder is 0
```

Comment:
2. 👍 
3. It is asking for you to state all the values that will be presented to the `console`.
4. 👍 
5. 👍 
6. It is asking here for you to say what the value of `a` is when `f1` is first called - so think about when `f1` is called for the first time - what arguments are passed to `a` the first time around ?
7. What will `a` be incremented to when `f2` is called for the first time - remember that `a` is incremented on each iteration of the `for` loop